src/routes/index.ts

```diff
@@ -17,3 +17,4 @@ export { default as TradingLeague } from './TradingLeague/TradingLeague';
 export { default as Profile } from './Profile/Profile';
 export { default as ProfileV1 } from './ProfileV1/Profile';
 export { default as Ecosystem } from './Ecosystem/Ecosystem';
+export { default as EcosystemV1 } from './EcosystemV1/Ecosystem';
```

Comment:
this will be current LP provider. we will work on the new one in the Ecosystem

EcosystemV1 => old
Ecosystem => new where Costin should work 

idea is to have a way to easily revert to old UI if something goes wrong through an ENV variable

Reply:
I see. Is the plan to then remove the EcosystemV1 once we have confirmed everything is working fine?

Reply:
yeah, we do same thing for Profile as of now. So main is showing ProfileV1 but we still develop community v2 features

Reply:
okay that seems okay to me